### ISSUE TITLE
[JENKINS-43885] Email-ext plugin throws dependency error

### ISSUE THREAD

**Issue by jenkins-infra-bot** [jenkinsci/email-ext-plugin#1224]
<p>{code}</p>

<p>27-Apr-2017 09:49:21.464 WARNING <span class="error">&#91;Initializing plugin copyartifact&#93;</span> hudson.ExtensionFinder$GuiceFinder$FaultTolerantScope$1.error Failed to instantiate Key[type=hudson.plugins.emailext.ExtendedEmailPublisherDescriptor, annotation=<span class="error">&#91;none&#93;</span>]; skipping this component<br/>
 com.google.inject.ProvisionException: Unable to provision, see the following errors:</p>

<p>1) Tried proxying hudson.plugins.emailext.ExtendedEmailPublisherDescriptor to support a circular dependency, but it is not an interface.</p>

<p>1 error<br/>
 at com.google.inject.internal.ProviderToInternalFactoryAdapter.get(ProviderToInternalFactoryAdapter.java:52)<br/>
 at com.google.inject.internal.SingletonScope$1.get(SingletonScope.java:145)<br/>
 at hudson.ExtensionFinder$GuiceFinder$FaultTolerantScope$1.get(ExtensionFinder.java:424)<br/>
 at com.google.inject.internal.InternalFactoryToProviderAdapter.get(InternalFactoryToProviderAdapter.java:41)<br/>
 at com.google.inject.internal.InjectorImpl$2$1.call(InjectorImpl.java:1016)<br/>
 at com.google.inject.internal.InjectorImpl.callInContext(InjectorImpl.java:1103)<br/>
 at com.google.inject.internal.InjectorImpl$2.get(InjectorImpl.java:1012)<br/>
 at hudson.ExtensionFinder$GuiceFinder._find(ExtensionFinder.java:386)<br/>
 at hudson.ExtensionFinder$GuiceFinder.find(ExtensionFinder.java:377)<br/>
 at hudson.ClassicPluginStrategy.findComponents(ClassicPluginStrategy.java:475)<br/>
 at hudson.ExtensionList.load(ExtensionList.java:365)<br/>
 at hudson.ExtensionList.ensureLoaded(ExtensionList.java:303)<br/>
 at hudson.ExtensionList.getComponents(ExtensionList.java:168)<br/>
 at hudson.DescriptorExtensionList.load(DescriptorExtensionList.java:191)<br/>
 at hudson.ExtensionList.ensureLoaded(ExtensionList.java:303)<br/>
 at hudson.ExtensionList.iterator(ExtensionList.java:157)<br/>
 at hudson.model.User.load(User.java:201)<br/>
 at hudson.model.User.<init>(User.java:155)<br/>
 at hudson.model.User.getOrCreate(User.java:464)<br/>
 at hudson.model.User.getById(User.java:535)<br/>
 at hudson.model.User.get(User.java:519)<br/>
 at hudson.model.User.current(User.java:503)<br/>
 at org.jenkinsci.plugins.scriptsecurity.scripts.ApprovalContext.withCurrentUser(ApprovalContext.java:73)<br/>
 at hudson.plugins.emailext.ExtendedEmailPublisherDescriptor.setDefaultPostsendScript(ExtendedEmailPublisherDescriptor.java:514)<br/>
 at hudson.plugins.emailext.ExtendedEmailPublisherDescriptor.<init>(ExtendedEmailPublisherDescriptor.java:183)<br/>
 at hudson.plugins.emailext.ExtendedEmailPublisherDescriptor$$FastClassByGuice$$5dfa54d0.newInstance(<generated>)<br/>
 at com.google.inject.internal.cglib.reflect.$FastConstructor.newInstance(FastConstructor.java:40)<br/>
 at com.google.inject.internal.DefaultConstructionProxyFactory$1.newInstance(DefaultConstructionProxyFactory.java:61)<br/>
 at com.google.inject.internal.ConstructorInjector.provision(ConstructorInjector.java:105)<br/>
 at com.google.inject.internal.ConstructorInjector.construct(ConstructorInjector.java:85)<br/>
 at com.google.inject.internal.ConstructorBindingImpl$Factory.get(ConstructorBindingImpl.java:267)<br/>
 at com.google.inject.internal.ProviderToInternalFactoryAdapter$1.call(ProviderToInternalFactoryAdapter.java:46)<br/>
 at com.google.inject.internal.InjectorImpl.callInContext(InjectorImpl.java:1103)<br/>
 at com.google.inject.internal.ProviderToInternalFactoryAdapter.get(ProviderToInternalFactoryAdapter.java:40)<br/>
 at com.google.inject.internal.SingletonScope$1.get(SingletonScope.java:145)<br/>
 at hudson.ExtensionFinder$GuiceFinder$FaultTolerantScope$1.get(ExtensionFinder.java:424)<br/>
 at com.google.inject.internal.InternalFactoryToProviderAdapter.get(InternalFactoryToProviderAdapter.java:41)<br/>
 at com.google.inject.internal.InjectorImpl$2$1.call(InjectorImpl.java:1016)<br/>
 at com.google.inject.internal.InjectorImpl.callInContext(InjectorImpl.java:1092)<br/>
 at com.google.inject.internal.InjectorImpl$2.get(InjectorImpl.java:1012)<br/>
 at hudson.ExtensionFinder$GuiceFinder._find(ExtensionFinder.java:386)<br/>
 at hudson.ExtensionFinder$GuiceFinder.find(ExtensionFinder.java:377)<br/>
 at hudson.ClassicPluginStrategy.findComponents(ClassicPluginStrategy.java:475)<br/>
 at hudson.ExtensionList.load(ExtensionList.java:365)<br/>
 at hudson.ExtensionList.ensureLoaded(ExtensionList.java:303)<br/>
 at hudson.ExtensionList.iterator(ExtensionList.java:157)<br/>
 at jenkins.model.Jenkins.getDescriptorByType(Jenkins.java:1544)<br/>
 at hudson.plugins.copyartifact.BuildSelectorParameter.initAliases(BuildSelectorParameter.java:156)<br/>
 at hudson.plugins.copyartifact.CopyArtifactPlugin.postInitialize(CopyArtifactPlugin.java:35)<br/>
 at hudson.PluginManager$2$1$2.run(PluginManager.java:542)<br/>
 at org.jvnet.hudson.reactor.TaskGraphBuilder$TaskImpl.run(TaskGraphBuilder.java:169)<br/>
 at org.jvnet.hudson.reactor.Reactor.runTask(Reactor.java:282)<br/>
 at jenkins.model.Jenkins$7.runTask(Jenkins.java:1088)<br/>
 at org.jvnet.hudson.reactor.Reactor$2.run(Reactor.java:210)<br/>
 at org.jvnet.hudson.reactor.Reactor$Node.run(Reactor.java:117)<br/>
 at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)<br/>
 at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)<br/>
 at java.lang.Thread.run(Thread.java:745)</p>

<p>{code}</p>

---
<details><summary><i>Originally reported by <a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=vikulin">vikulin</a>, imported from: <a class="original-jira-link" href="https://issues.jenkins.io/browse/JENKINS-43885" target="_blank">Email-ext plugin throws dependency error</a></i></summary>
<i><ul>
<li><b>assignee</b>: <img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/slide_o_mix.png" title="slide_o_mix's avatar" /> <a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=slide_o_mix">slide_o_mix</a>
<li><b>status</b>: Closed
<li><b>priority</b>: Major
<li><b>component(s)</b>: email-ext-plugin
<li><b>resolution</b>: Incomplete
<li><b>resolved</b>: 2019-03-22T13:49:32+00:00
<li><b>votes</b>: 1
<li><b>watchers</b>: 6
<li><b>imported</b>: 20251211-193728
</ul></i>
<details><summary>Raw content of original issue</summary>

<pre>
<p>{code}</p>

<p>27-Apr-2017 09:49:21.464 WARNING <span class="error">&#91;Initializing plugin copyartifact&#93;</span> hudson.ExtensionFinder$GuiceFinder$FaultTolerantScope$1.error Failed to instantiate Key[type=hudson.plugins.emailext.ExtendedEmailPublisherDescriptor, annotation=<span class="error">&#91;none&#93;</span>]; skipping this component
 com.google.inject.ProvisionException: Unable to provision, see the following errors:</p>

<p>1) Tried proxying hudson.plugins.emailext.ExtendedEmailPublisherDescriptor to support a circular dependency, but it is not an interface.</p>

<p>1 error
 at com.google.inject.internal.ProviderToInternalFactoryAdapter.get(ProviderToInternalFactoryAdapter.java:52)
 at com.google.inject.internal.SingletonScope$1.get(SingletonScope.java:145)
 at hudson.ExtensionFinder$GuiceFinder$FaultTolerantScope$1.get(ExtensionFinder.java:424)
 at com.google.inject.internal.InternalFactoryToProviderAdapter.get(InternalFactoryToProviderAdapter.java:41)
 at com.google.inject.internal.InjectorImpl$2$1.call(InjectorImpl.java:1016)
 at com.google.inject.internal.InjectorImpl.callInContext(InjectorImpl.java:1103)
 at com.google.inject.internal.InjectorImpl$2.get(InjectorImpl.java:1012)
 at hudson.ExtensionFinder$GuiceFinder._find(ExtensionFinder.java:386)
 at hudson.ExtensionFinder$GuiceFinder.find(ExtensionFinder.java:377)
 at hudson.ClassicPluginStrategy.findComponents(ClassicPluginStrategy.java:475)
 at hudson.ExtensionList.load(ExtensionList.java:365)
 at hudson.ExtensionList.ensureLoaded(ExtensionList.java:303)
 at hudson.ExtensionList.getComponents(ExtensionList.java:168)
 at hudson.DescriptorExtensionList.load(DescriptorExtensionList.java:191)
 at hudson.ExtensionList.ensureLoaded(ExtensionList.java:303)
 at hudson.ExtensionList.iterator(ExtensionList.java:157)
 at hudson.model.User.load(User.java:201)
 at hudson.model.User.&lt;init&gt;(User.java:155)
 at hudson.model.User.getOrCreate(User.java:464)
 at hudson.model.User.getById(User.java:535)
 at hudson.model.User.get(User.java:519)
 at hudson.model.User.current(User.java:503)
 at org.jenkinsci.plugins.scriptsecurity.scripts.ApprovalContext.withCurrentUser(ApprovalContext.java:73)
 at hudson.plugins.emailext.ExtendedEmailPublisherDescriptor.setDefaultPostsendScript(ExtendedEmailPublisherDescriptor.java:514)
 at hudson.plugins.emailext.ExtendedEmailPublisherDescriptor.&lt;init&gt;(ExtendedEmailPublisherDescriptor.java:183)
 at hudson.plugins.emailext.ExtendedEmailPublisherDescriptor$$FastClassByGuice$$5dfa54d0.newInstance(&lt;generated&gt;)
 at com.google.inject.internal.cglib.reflect.$FastConstructor.newInstance(FastConstructor.java:40)
 at com.google.inject.internal.DefaultConstructionProxyFactory$1.newInstance(DefaultConstructionProxyFactory.java:61)
 at com.google.inject.internal.ConstructorInjector.provision(ConstructorInjector.java:105)
 at com.google.inject.internal.ConstructorInjector.construct(ConstructorInjector.java:85)
 at com.google.inject.internal.ConstructorBindingImpl$Factory.get(ConstructorBindingImpl.java:267)
 at com.google.inject.internal.ProviderToInternalFactoryAdapter$1.call(ProviderToInternalFactoryAdapter.java:46)
 at com.google.inject.internal.InjectorImpl.callInContext(InjectorImpl.java:1103)
 at com.google.inject.internal.ProviderToInternalFactoryAdapter.get(ProviderToInternalFactoryAdapter.java:40)
 at com.google.inject.internal.SingletonScope$1.get(SingletonScope.java:145)
 at hudson.ExtensionFinder$GuiceFinder$FaultTolerantScope$1.get(ExtensionFinder.java:424)
 at com.google.inject.internal.InternalFactoryToProviderAdapter.get(InternalFactoryToProviderAdapter.java:41)
 at com.google.inject.internal.InjectorImpl$2$1.call(InjectorImpl.java:1016)
 at com.google.inject.internal.InjectorImpl.callInContext(InjectorImpl.java:1092)
 at com.google.inject.internal.InjectorImpl$2.get(InjectorImpl.java:1012)
 at hudson.ExtensionFinder$GuiceFinder._find(ExtensionFinder.java:386)
 at hudson.ExtensionFinder$GuiceFinder.find(ExtensionFinder.java:377)
 at hudson.ClassicPluginStrategy.findComponents(ClassicPluginStrategy.java:475)
 at hudson.ExtensionList.load(ExtensionList.java:365)
 at hudson.ExtensionList.ensureLoaded(ExtensionList.java:303)
 at hudson.ExtensionList.iterator(ExtensionList.java:157)
 at jenkins.model.Jenkins.getDescriptorByType(Jenkins.java:1544)
 at hudson.plugins.copyartifact.BuildSelectorParameter.initAliases(BuildSelectorParameter.java:156)
 at hudson.plugins.copyartifact.CopyArtifactPlugin.postInitialize(CopyArtifactPlugin.java:35)
 at hudson.PluginManager$2$1$2.run(PluginManager.java:542)
 at org.jvnet.hudson.reactor.TaskGraphBuilder$TaskImpl.run(TaskGraphBuilder.java:169)
 at org.jvnet.hudson.reactor.Reactor.runTask(Reactor.java:282)
 at jenkins.model.Jenkins$7.runTask(Jenkins.java:1088)
 at org.jvnet.hudson.reactor.Reactor$2.run(Reactor.java:210)
 at org.jvnet.hudson.reactor.Reactor$Node.run(Reactor.java:117)
 at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
 at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
 at java.lang.Thread.run(Thread.java:745)</p>

<p>{code}</p></pre>
</details>
</details>
<ul><li><i>environment</i>: <code>Jenkins 2.55,2.56,2.57, email ext plugin 2.57.2</code></li></ul>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i><img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/chandau.png" title="chandau's avatar" /> chandau:</i></summary>

<ul>
<li><i>Original <a class="original-jira-link" href="https://issues.jenkins.io/browse/JENKINS-43885?focusedId=336293&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-336293">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p>still seeing this on Jenkins 2.107.2 and email-ext 2.62</p>
</pre>
</details>

</ul>
</details>

<p>still seeing this on Jenkins 2.107.2 and email-ext 2.62</p>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i>alex_vol:</i></summary>

<ul>
<li><i>Original <a class="original-jira-link" href="https://issues.jenkins.io/browse/JENKINS-43885?focusedId=343276&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-343276">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p>I tracked this to changes in CopyArtifact plugin, it may have nothing to do with email-ext but more related to the recently changed CopyArtifact. At least that is what my initial investigation is leading me to. I will downgrade CopyArtifact to 1.40 or 1.39.1 to verify this.</p>
</pre>
</details>

</ul>
</details>

<p>I tracked this to changes in CopyArtifact plugin, it may have nothing to do with email-ext but more related to the recently changed CopyArtifact. At least that is what my initial investigation is leading me to. I will downgrade CopyArtifact to 1.40 or 1.39.1 to verify this.</p>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i>andreip:</i></summary>

<ul>
<li><i>Original <a class="original-jira-link" href="https://issues.jenkins.io/browse/JENKINS-43885?focusedId=345711&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-345711">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p>Seeing this on Jenkins ver. 2.136. This is my list of plugins. Please share yours if you have this issue:</p>

<p> </p>

<p>Amazon Web Services SDK 1.11.341
AnsiColor 0.5.2
Ant Plugin 1.8
Apache HttpComponents Client 4.x API Plugin 4.5.5-3.0
Authentication Tokens API Plugin 1.3 
Autofavorite for Blue Ocean 1.2.2
Bitbucket Branch Source Plugin 2.2.12
Bitbucket Build Status Notifier Plugin 1.4.0
Bitbucket Pipeline for Blue Ocean 1.7.2
Bitbucket Plugin 1.1.8
Blue Ocean 1.7.2
Blue Ocean Core JS 1.7.2
Blue Ocean Pipeline Editor 1.7.2
bouncycastle API Plugin 2.16.3
Branch API Plugin 2.0.20
Build Failure Analyzer 1.20.0
Build Timeout 1.19
Claim Plugin 2.15
CloudBees Amazon Web Services Credentials Plugin 1.23
Cobertura Plugin 1.12.1
Command Agent Launcher Plugin 1.2
Common API for Blue Ocean 1.7.2
Conditional BuildStep 1.3.6
Config API for Blue Ocean 1.7.2
Config File Provider Plugin 2.18
Configuration Slicing plugin 1.47
Credentials Binding Plugin 1.16
Credentials Plugin 2.1.18
Dashboard for Blue Ocean 1.7.2
Dashboard View 2.9.11
Design Language 1.7.2
Display URL API 2.2.0
Display URL for Blue Ocean 2.2.0
Docker Commons Plugin 1.13
Docker Pipeline 1.17
Durable Task Plugin 1.24
EC2 Fleet Jenkins Plugin 1.1.8-SNAPSHOT (private-b9f3f4cb-cyrilb)
Email Extension Plugin 2.63
Events API for Blue Ocean 1.7.2
External Monitor Job Type Plugin 1.7
Favorite 2.3.2
Folders Plugin 6.5.1
Git client plugin 2.7.3
Git Pipeline for Blue Ocean 1.7.2
Git plugin 3.9.1
GIT server Plugin 1.7 
GitHub API Plugin 1.92
GitHub Branch Source Plugin 2.3.6
GitHub Pipeline for Blue Ocean 1.7.2
GitHub plugin 1.29.2
Gradle Plugin 1.29
Green Balls 1.15 
Handy Uri Templates 2.x API Plugin 2.1.6-1.0 
HipChat Plugin 2.1.1 
HTML Publisher plugin 1.16
i18n for Blue Ocean 1.7.2
Icon Shim Plugin 2.0.3 
Jackson 2 API Plugin 2.8.11.3
Javadoc Plugin 1.4
JavaScript GUI Lib: ACE Editor bundle plugin 1.1 
JavaScript GUI Lib: Handlebars bundle plugin 1.1.1 
JavaScript GUI Lib: jQuery bundles (jQuery and jQuery UI) plugin 1.2.1 
JavaScript GUI Lib: Moment.js bundle plugin 1.1.1 
JDK Tool Plugin 1.1
JIRA Integration for Blue Ocean 1.7.2
JIRA plugin 3.0.0
Job DSL 1.70
jQuery plugin 1.12.4-0
JSch dependency plugin 0.1.54.2
JUnit Plugin 1.24
JWT for Blue Ocean 1.7.2
LDAP Plugin 1.20
Lockable Resources plugin 2.3
Mailer Plugin 1.21
MapDB API Plugin 1.0.9.0
Matrix Authorization Strategy Plugin 2.3
Matrix Project Plugin 1.13
Maven Integration plugin 3.1.2
Maven Release Plug-in Plug-in 0.14.0 
Mercurial plugin 2.4
Metrics Plugin 4.0.2.2
Multiple SCMs plugin 0.6 
Naginator 1.17.2 
Node Iterator API Plugin 1.5.0 
OWASP Dependency-Check Plugin 3.3.1
OWASP Markup Formatter Plugin 1.5
PAM Authentication plugin 1.3
Parameterized Trigger plugin 2.35.2
Personalization for Blue Ocean 1.7.2
Pipeline 2.5
Pipeline Graph Analysis Plugin 1.7
Pipeline implementation for Blue Ocean 1.7.2
Pipeline Maven Integration Plugin 3.5.11
Pipeline NPM Integration Plugin 0.9.2
Pipeline SCM API for Blue Ocean 1.7.2
Pipeline Utility Steps 2.1.0
Pipeline: API 2.29
Pipeline: Basic Steps 2.9
Pipeline: Build Step 2.7
Pipeline: Declarative 1.3.1
Pipeline: Declarative Agent API 1.1.1 
Pipeline: Declarative Extension Points API 1.3.1
Pipeline: Groovy 2.54
Pipeline: Input Step 2.8
Pipeline: Job 2.24
Pipeline: Milestone Step 1.3.1
Pipeline: Model API 1.3.1
Pipeline: Multibranch 2.20
Pipeline: Nodes and Processes 2.20
Pipeline: REST API Plugin 2.10
Pipeline: SCM Step 2.6
Pipeline: Shared Groovy Libraries 2.9
Pipeline: Stage Step 2.3
Pipeline: Stage Tags Metadata 1.3.1
Pipeline: Stage View Plugin 2.10
Pipeline: Step API 2.16
Pipeline: Supporting APIs 2.20
Plain Credentials Plugin 1.4
Pub-Sub "light" Bus 1.12
Release Plugin 2.10.1
REST API for Blue Ocean 1.7.2
REST Implementation for Blue Ocean 1.7.2
Run Condition Plugin 1.0 
SCM API Plugin 2.2.7
Script Security Plugin 1.44
Server Sent Events (SSE) Gateway Plugin 1.15 
SonarQube Scanner for Jenkins 2.8
SSH Agent Plugin 1.16
SSH Credentials Plugin 1.14
SSH Slaves plugin 1.26
Static Analysis Utilities 1.95
Structs Plugin 1.14
Subversion Plug-in 2.11.1
Test stability history 2.3
Token Macro Plugin 2.5
user build vars plugin 1.5 
Variant Plugin 1.1 
Warnings Plug-in 4.68
Web for Blue Ocean 1.7.2
Windows Slaves Plugin 1.3.1</p>
</pre>
</details>

</ul>
</details>

<p>Seeing this on Jenkins ver. 2.136. This is my list of plugins. Please share yours if you have this issue:</p>

<p> </p>

<p>Amazon Web Services SDK 1.11.341<br/>
AnsiColor 0.5.2<br/>
Ant Plugin 1.8<br/>
Apache HttpComponents Client 4.x API Plugin 4.5.5-3.0<br/>
Authentication Tokens API Plugin 1.3 <br/>
Autofavorite for Blue Ocean 1.2.2<br/>
Bitbucket Branch Source Plugin 2.2.12<br/>
Bitbucket Build Status Notifier Plugin 1.4.0<br/>
Bitbucket Pipeline for Blue Ocean 1.7.2<br/>
Bitbucket Plugin 1.1.8<br/>
Blue Ocean 1.7.2<br/>
Blue Ocean Core JS 1.7.2<br/>
Blue Ocean Pipeline Editor 1.7.2<br/>
bouncycastle API Plugin 2.16.3<br/>
Branch API Plugin 2.0.20<br/>
Build Failure Analyzer 1.20.0<br/>
Build Timeout 1.19<br/>
Claim Plugin 2.15<br/>
CloudBees Amazon Web Services Credentials Plugin 1.23<br/>
Cobertura Plugin 1.12.1<br/>
Command Agent Launcher Plugin 1.2<br/>
Common API for Blue Ocean 1.7.2<br/>
Conditional BuildStep 1.3.6<br/>
Config API for Blue Ocean 1.7.2<br/>
Config File Provider Plugin 2.18<br/>
Configuration Slicing plugin 1.47<br/>
Credentials Binding Plugin 1.16<br/>
Credentials Plugin 2.1.18<br/>
Dashboard for Blue Ocean 1.7.2<br/>
Dashboard View 2.9.11<br/>
Design Language 1.7.2<br/>
Display URL API 2.2.0<br/>
Display URL for Blue Ocean 2.2.0<br/>
Docker Commons Plugin 1.13<br/>
Docker Pipeline 1.17<br/>
Durable Task Plugin 1.24<br/>
EC2 Fleet Jenkins Plugin 1.1.8-SNAPSHOT (private-b9f3f4cb-cyrilb)<br/>
Email Extension Plugin 2.63<br/>
Events API for Blue Ocean 1.7.2<br/>
External Monitor Job Type Plugin 1.7<br/>
Favorite 2.3.2<br/>
Folders Plugin 6.5.1<br/>
Git client plugin 2.7.3<br/>
Git Pipeline for Blue Ocean 1.7.2<br/>
Git plugin 3.9.1<br/>
GIT server Plugin 1.7 <br/>
GitHub API Plugin 1.92<br/>
GitHub Branch Source Plugin 2.3.6<br/>
GitHub Pipeline for Blue Ocean 1.7.2<br/>
GitHub plugin 1.29.2<br/>
Gradle Plugin 1.29<br/>
Green Balls 1.15 <br/>
Handy Uri Templates 2.x API Plugin 2.1.6-1.0 <br/>
HipChat Plugin 2.1.1 <br/>
HTML Publisher plugin 1.16<br/>
i18n for Blue Ocean 1.7.2<br/>
Icon Shim Plugin 2.0.3 <br/>
Jackson 2 API Plugin 2.8.11.3<br/>
Javadoc Plugin 1.4<br/>
JavaScript GUI Lib: ACE Editor bundle plugin 1.1 <br/>
JavaScript GUI Lib: Handlebars bundle plugin 1.1.1 <br/>
JavaScript GUI Lib: jQuery bundles (jQuery and jQuery UI) plugin 1.2.1 <br/>
JavaScript GUI Lib: Moment.js bundle plugin 1.1.1 <br/>
JDK Tool Plugin 1.1<br/>
JIRA Integration for Blue Ocean 1.7.2<br/>
JIRA plugin 3.0.0<br/>
Job DSL 1.70<br/>
jQuery plugin 1.12.4-0<br/>
JSch dependency plugin 0.1.54.2<br/>
JUnit Plugin 1.24<br/>
JWT for Blue Ocean 1.7.2<br/>
LDAP Plugin 1.20<br/>
Lockable Resources plugin 2.3<br/>
Mailer Plugin 1.21<br/>
MapDB API Plugin 1.0.9.0<br/>
Matrix Authorization Strategy Plugin 2.3<br/>
Matrix Project Plugin 1.13<br/>
Maven Integration plugin 3.1.2<br/>
Maven Release Plug-in Plug-in 0.14.0 <br/>
Mercurial plugin 2.4<br/>
Metrics Plugin 4.0.2.2<br/>
Multiple SCMs plugin 0.6 <br/>
Naginator 1.17.2 <br/>
Node Iterator API Plugin 1.5.0 <br/>
OWASP Dependency-Check Plugin 3.3.1<br/>
OWASP Markup Formatter Plugin 1.5<br/>
PAM Authentication plugin 1.3<br/>
Parameterized Trigger plugin 2.35.2<br/>
Personalization for Blue Ocean 1.7.2<br/>
Pipeline 2.5<br/>
Pipeline Graph Analysis Plugin 1.7<br/>
Pipeline implementation for Blue Ocean 1.7.2<br/>
Pipeline Maven Integration Plugin 3.5.11<br/>
Pipeline NPM Integration Plugin 0.9.2<br/>
Pipeline SCM API for Blue Ocean 1.7.2<br/>
Pipeline Utility Steps 2.1.0<br/>
Pipeline: API 2.29<br/>
Pipeline: Basic Steps 2.9<br/>
Pipeline: Build Step 2.7<br/>
Pipeline: Declarative 1.3.1<br/>
Pipeline: Declarative Agent API 1.1.1 <br/>
Pipeline: Declarative Extension Points API 1.3.1<br/>
Pipeline: Groovy 2.54<br/>
Pipeline: Input Step 2.8<br/>
Pipeline: Job 2.24<br/>
Pipeline: Milestone Step 1.3.1<br/>
Pipeline: Model API 1.3.1<br/>
Pipeline: Multibranch 2.20<br/>
Pipeline: Nodes and Processes 2.20<br/>
Pipeline: REST API Plugin 2.10<br/>
Pipeline: SCM Step 2.6<br/>
Pipeline: Shared Groovy Libraries 2.9<br/>
Pipeline: Stage Step 2.3<br/>
Pipeline: Stage Tags Metadata 1.3.1<br/>
Pipeline: Stage View Plugin 2.10<br/>
Pipeline: Step API 2.16<br/>
Pipeline: Supporting APIs 2.20<br/>
Plain Credentials Plugin 1.4<br/>
Pub-Sub "light" Bus 1.12<br/>
Release Plugin 2.10.1<br/>
REST API for Blue Ocean 1.7.2<br/>
REST Implementation for Blue Ocean 1.7.2<br/>
Run Condition Plugin 1.0 <br/>
SCM API Plugin 2.2.7<br/>
Script Security Plugin 1.44<br/>
Server Sent Events (SSE) Gateway Plugin 1.15 <br/>
SonarQube Scanner for Jenkins 2.8<br/>
SSH Agent Plugin 1.16<br/>
SSH Credentials Plugin 1.14<br/>
SSH Slaves plugin 1.26<br/>
Static Analysis Utilities 1.95<br/>
Structs Plugin 1.14<br/>
Subversion Plug-in 2.11.1<br/>
Test stability history 2.3<br/>
Token Macro Plugin 2.5<br/>
user build vars plugin 1.5 <br/>
Variant Plugin 1.1 <br/>
Warnings Plug-in 4.68<br/>
Web for Blue Ocean 1.7.2<br/>
Windows Slaves Plugin 1.3.1</p>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i>andreip:</i></summary>

<ul>
<li><i>Original <a class="original-jira-link" href="https://issues.jenkins.io/browse/JENKINS-43885?focusedId=345713&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-345713">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p>My stack trace is slighty different</p>



<p>Aug 08, 2018 10:29:30 AM hudson.ExtensionFinder$GuiceFinder$FaultTolerantScope$1 error
WARNING: Failed to instantiate Key[type=hudson.plugins.emailext.ExtendedEmailPublisherDescriptor, annotation=<span class="error">&#91;none&#93;</span>]; skipping this component
com.google.inject.ProvisionException: Unable to provision, see the following errors:</p>

<p>1) Tried proxying hudson.plugins.emailext.ExtendedEmailPublisherDescriptor to support a circular dependency, but it is not an interface.</p>

<p>1 error
 at com.google.inject.internal.ProviderToInternalFactoryAdapter.get(ProviderToInternalFactoryAdapter.java:52)
 at com.google.inject.internal.SingletonScope$1.get(SingletonScope.java:145)
 at hudson.ExtensionFinder$GuiceFinder$FaultTolerantScope$1.get(ExtensionFinder.java:432)
 at com.google.inject.internal.InternalFactoryToProviderAdapter.get(InternalFactoryToProviderAdapter.java:41)
 at com.google.inject.internal.InjectorImpl$2$1.call(InjectorImpl.java:1016)
 at com.google.inject.internal.InjectorImpl.callInContext(InjectorImpl.java:1103)
 at com.google.inject.internal.InjectorImpl$2.get(InjectorImpl.java:1012)
 at hudson.ExtensionFinder$GuiceFinder._find(ExtensionFinder.java:394)
 at hudson.ExtensionFinder$GuiceFinder.find(ExtensionFinder.java:385)
 at hudson.ClassicPluginStrategy.findComponents(ClassicPluginStrategy.java:493)
 at hudson.ExtensionList.load(ExtensionList.java:380)
 at hudson.ExtensionList.ensureLoaded(ExtensionList.java:318)
 at hudson.ExtensionList.getComponents(ExtensionList.java:183)
 at hudson.DescriptorExtensionList.load(DescriptorExtensionList.java:192)
 at hudson.ExtensionList.ensureLoaded(ExtensionList.java:318)
 at hudson.ExtensionList.iterator(ExtensionList.java:172)
 at hudson.model.User.load(User.java:207)
 at hudson.model.User.&lt;init&gt;(User.java:161)
 at hudson.model.User.getOrCreate(User.java:532)
 at hudson.model.User.getOrCreate(User.java:479)
 at hudson.model.User.getById(User.java:668)
 at hudson.model.User.get(User.java:651)
 at hudson.model.User.current(User.java:635)
 at org.jenkinsci.plugins.scriptsecurity.scripts.ApprovalContext.withCurrentUser(ApprovalContext.java:72)
 at hudson.plugins.emailext.ExtendedEmailPublisherDescriptor.setDefaultPostsendScript(ExtendedEmailPublisherDescriptor.java:575)
 at hudson.plugins.emailext.ExtendedEmailPublisherDescriptor.&lt;init&gt;(ExtendedEmailPublisherDescriptor.java:191)
 at hudson.plugins.emailext.ExtendedEmailPublisherDescriptor$$FastClassByGuice$$5dfa54d0.newInstance(&lt;generated&gt;)
 at com.google.inject.internal.cglib.reflect.$FastConstructor.newInstance(FastConstructor.java:40)
 at com.google.inject.internal.DefaultConstructionProxyFactory$1.newInstance(DefaultConstructionProxyFactory.java:61)
 at com.google.inject.internal.ConstructorInjector.provision(ConstructorInjector.java:105)
 at com.google.inject.internal.ConstructorInjector.construct(ConstructorInjector.java:85)
 at com.google.inject.internal.ConstructorBindingImpl$Factory.get(ConstructorBindingImpl.java:267)
 at com.google.inject.internal.ProviderToInternalFactoryAdapter$1.call(ProviderToInternalFactoryAdapter.java:46)
 at com.google.inject.internal.InjectorImpl.callInContext(InjectorImpl.java:1103)
 at com.google.inject.internal.ProviderToInternalFactoryAdapter.get(ProviderToInternalFactoryAdapter.java:40)
 at com.google.inject.internal.SingletonScope$1.get(SingletonScope.java:145)
 at hudson.ExtensionFinder$GuiceFinder$FaultTolerantScope$1.get(ExtensionFinder.java:432)
 at com.google.inject.internal.InternalFactoryToProviderAdapter.get(InternalFactoryToProviderAdapter.java:41)
 at com.google.inject.internal.InjectorImpl$2$1.call(InjectorImpl.java:1016)
 at com.google.inject.internal.InjectorImpl.callInContext(InjectorImpl.java:1092)
 at com.google.inject.internal.InjectorImpl$2.get(InjectorImpl.java:1012)
 at hudson.ExtensionFinder$GuiceFinder._find(ExtensionFinder.java:394)
 at hudson.ExtensionFinder$GuiceFinder.find(ExtensionFinder.java:385)
 at hudson.ClassicPluginStrategy.findComponents(ClassicPluginStrategy.java:493)
 at hudson.ExtensionList.load(ExtensionList.java:380)
 at hudson.ExtensionList.ensureLoaded(ExtensionList.java:318)
 at hudson.ExtensionList.iterator(ExtensionList.java:172)
 at jenkins.model.Jenkins.getDescriptorByType(Jenkins.java:1522)
 at hudson.plugins.git.GitSCM.onLoaded(GitSCM.java:1887)
 at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
 at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
 at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
 at java.lang.reflect.Method.invoke(Method.java:498)
 at hudson.init.TaskMethodFinder.invoke(TaskMethodFinder.java:104)
 at hudson.init.TaskMethodFinder$TaskImpl.run(TaskMethodFinder.java:175)
 at org.jvnet.hudson.reactor.Reactor.runTask(Reactor.java:296)
 at jenkins.model.Jenkins$5.runTask(Jenkins.java:1066)
 at org.jvnet.hudson.reactor.Reactor$2.run(Reactor.java:214)
 at org.jvnet.hudson.reactor.Reactor$Node.run(Reactor.java:117)
 at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
 at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
 at java.lang.Thread.run(Thread.java:748)</p>
</pre>
</details>

</ul>
</details>

<p>My stack trace is slighty different</p>



<p>Aug 08, 2018 10:29:30 AM hudson.ExtensionFinder$GuiceFinder$FaultTolerantScope$1 error<br/>
WARNING: Failed to instantiate Key[type=hudson.plugins.emailext.ExtendedEmailPublisherDescriptor, annotation=<span class="error">&#91;none&#93;</span>]; skipping this component<br/>
com.google.inject.ProvisionException: Unable to provision, see the following errors:</p>

<p>1) Tried proxying hudson.plugins.emailext.ExtendedEmailPublisherDescriptor to support a circular dependency, but it is not an interface.</p>

<p>1 error<br/>
 at com.google.inject.internal.ProviderToInternalFactoryAdapter.get(ProviderToInternalFactoryAdapter.java:52)<br/>
 at com.google.inject.internal.SingletonScope$1.get(SingletonScope.java:145)<br/>
 at hudson.ExtensionFinder$GuiceFinder$FaultTolerantScope$1.get(ExtensionFinder.java:432)<br/>
 at com.google.inject.internal.InternalFactoryToProviderAdapter.get(InternalFactoryToProviderAdapter.java:41)<br/>
 at com.google.inject.internal.InjectorImpl$2$1.call(InjectorImpl.java:1016)<br/>
 at com.google.inject.internal.InjectorImpl.callInContext(InjectorImpl.java:1103)<br/>
 at com.google.inject.internal.InjectorImpl$2.get(InjectorImpl.java:1012)<br/>
 at hudson.ExtensionFinder$GuiceFinder._find(ExtensionFinder.java:394)<br/>
 at hudson.ExtensionFinder$GuiceFinder.find(ExtensionFinder.java:385)<br/>
 at hudson.ClassicPluginStrategy.findComponents(ClassicPluginStrategy.java:493)<br/>
 at hudson.ExtensionList.load(ExtensionList.java:380)<br/>
 at hudson.ExtensionList.ensureLoaded(ExtensionList.java:318)<br/>
 at hudson.ExtensionList.getComponents(ExtensionList.java:183)<br/>
 at hudson.DescriptorExtensionList.load(DescriptorExtensionList.java:192)<br/>
 at hudson.ExtensionList.ensureLoaded(ExtensionList.java:318)<br/>
 at hudson.ExtensionList.iterator(ExtensionList.java:172)<br/>
 at hudson.model.User.load(User.java:207)<br/>
 at hudson.model.User.<init>(User.java:161)<br/>
 at hudson.model.User.getOrCreate(User.java:532)<br/>
 at hudson.model.User.getOrCreate(User.java:479)<br/>
 at hudson.model.User.getById(User.java:668)<br/>
 at hudson.model.User.get(User.java:651)<br/>
 at hudson.model.User.current(User.java:635)<br/>
 at org.jenkinsci.plugins.scriptsecurity.scripts.ApprovalContext.withCurrentUser(ApprovalContext.java:72)<br/>
 at hudson.plugins.emailext.ExtendedEmailPublisherDescriptor.setDefaultPostsendScript(ExtendedEmailPublisherDescriptor.java:575)<br/>
 at hudson.plugins.emailext.ExtendedEmailPublisherDescriptor.<init>(ExtendedEmailPublisherDescriptor.java:191)<br/>
 at hudson.plugins.emailext.ExtendedEmailPublisherDescriptor$$FastClassByGuice$$5dfa54d0.newInstance(<generated>)<br/>
 at com.google.inject.internal.cglib.reflect.$FastConstructor.newInstance(FastConstructor.java:40)<br/>
 at com.google.inject.internal.DefaultConstructionProxyFactory$1.newInstance(DefaultConstructionProxyFactory.java:61)<br/>
 at com.google.inject.internal.ConstructorInjector.provision(ConstructorInjector.java:105)<br/>
 at com.google.inject.internal.ConstructorInjector.construct(ConstructorInjector.java:85)<br/>
 at com.google.inject.internal.ConstructorBindingImpl$Factory.get(ConstructorBindingImpl.java:267)<br/>
 at com.google.inject.internal.ProviderToInternalFactoryAdapter$1.call(ProviderToInternalFactoryAdapter.java:46)<br/>
 at com.google.inject.internal.InjectorImpl.callInContext(InjectorImpl.java:1103)<br/>
 at com.google.inject.internal.ProviderToInternalFactoryAdapter.get(ProviderToInternalFactoryAdapter.java:40)<br/>
 at com.google.inject.internal.SingletonScope$1.get(SingletonScope.java:145)<br/>
 at hudson.ExtensionFinder$GuiceFinder$FaultTolerantScope$1.get(ExtensionFinder.java:432)<br/>
 at com.google.inject.internal.InternalFactoryToProviderAdapter.get(InternalFactoryToProviderAdapter.java:41)<br/>
 at com.google.inject.internal.InjectorImpl$2$1.call(InjectorImpl.java:1016)<br/>
 at com.google.inject.internal.InjectorImpl.callInContext(InjectorImpl.java:1092)<br/>
 at com.google.inject.internal.InjectorImpl$2.get(InjectorImpl.java:1012)<br/>
 at hudson.ExtensionFinder$GuiceFinder._find(ExtensionFinder.java:394)<br/>
 at hudson.ExtensionFinder$GuiceFinder.find(ExtensionFinder.java:385)<br/>
 at hudson.ClassicPluginStrategy.findComponents(ClassicPluginStrategy.java:493)<br/>
 at hudson.ExtensionList.load(ExtensionList.java:380)<br/>
 at hudson.ExtensionList.ensureLoaded(ExtensionList.java:318)<br/>
 at hudson.ExtensionList.iterator(ExtensionList.java:172)<br/>
 at jenkins.model.Jenkins.getDescriptorByType(Jenkins.java:1522)<br/>
 at hudson.plugins.git.GitSCM.onLoaded(GitSCM.java:1887)<br/>
 at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)<br/>
 at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)<br/>
 at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)<br/>
 at java.lang.reflect.Method.invoke(Method.java:498)<br/>
 at hudson.init.TaskMethodFinder.invoke(TaskMethodFinder.java:104)<br/>
 at hudson.init.TaskMethodFinder$TaskImpl.run(TaskMethodFinder.java:175)<br/>
 at org.jvnet.hudson.reactor.Reactor.runTask(Reactor.java:296)<br/>
 at jenkins.model.Jenkins$5.runTask(Jenkins.java:1066)<br/>
 at org.jvnet.hudson.reactor.Reactor$2.run(Reactor.java:214)<br/>
 at org.jvnet.hudson.reactor.Reactor$Node.run(Reactor.java:117)<br/>
 at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)<br/>
 at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)<br/>
 at java.lang.Thread.run(Thread.java:748)</p>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i><img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/davidvanlaatum.png" title="davidvanlaatum's avatar" /> davidvanlaatum:</i></summary>

<ul>
<li><i>Original <a class="original-jira-link" href="https://issues.jenkins.io/browse/JENKINS-43885?focusedId=345767&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-345767">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p>"Tried proxying hudson.plugins.emailext.ExtendedEmailPublisherDescriptor to support a circular dependency, but it is not an interface." is entirely valid this can't be an interface as its a descriptor and the backtrace gives no clues as to why, unless you have a post send script that somehow has a decency on something that in turn depends on email-ext but that seems unlikly</p>
</pre>
</details>

</ul>
</details>

<p>"Tried proxying hudson.plugins.emailext.ExtendedEmailPublisherDescriptor to support a circular dependency, but it is not an interface." is entirely valid this can't be an interface as its a descriptor and the backtrace gives no clues as to why, unless you have a post send script that somehow has a decency on something that in turn depends on email-ext but that seems unlikly</p>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i>andreip:</i></summary>

<ul>
<li><i>Original <a class="original-jira-link" href="https://issues.jenkins.io/browse/JENKINS-43885?focusedId=345868&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-345868">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p><a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=davidvanlaatum" class="user-hover" rel="davidvanlaatum">davidvanlaatum</a> we actually have a presendScript.</p>



<p><font color="#6a8759">presendScript</font>: <font color="#6a8759">"""</font><font color="#6a8759">List&lt;String&gt; domainWhiteList = <span class="error">&#91;&quot;@mydomain.com&quot;, &quot;@mydomain.nl&quot;&#93;</span></font><font color="#6a8759">List&lt;String&gt; containsBlackList = <span class="error">&#91;&quot;noreply&quot;&#93;</span></font><font color="#6a8759">def recipients = msg.getRecipients(javax.mail.Message.RecipientType.TO)</font><font color="#6a8759">def filtered = recipients.findAll { emailAddress -&gt;</font><font color="#6a8759"> def emailAddressString = emailAddress.toString()</font><font color="#6a8759"> return domainWhiteList.any { domain -&gt; emailAddressString.endsWith(domain) } &amp;&amp; </font><font color="#6a8759"> !containsBlackList.any { value -&gt; emailAddressString.contains(value) }</font>{color:#6a8759}}
<font color=""><font color="#6a8759"></font>msg.setRecipients(javax.mail.Message.RecipientType.TO, filtered as javax.mail.Address[])</font><font color="#6a8759">"""</font></p>
</pre>
</details>

</ul>
</details>

<p><a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=davidvanlaatum" class="user-hover" rel="davidvanlaatum">davidvanlaatum</a> we actually have a presendScript.</p>



<p><font color="#6a8759">presendScript</font>: <font color="#6a8759">"""</font><font color="#6a8759">List<String> domainWhiteList = <span class="error">&#91;"@​mydomain.com", "@​mydomain.nl"&#93;</span></font><font color="#6a8759">List<String> containsBlackList = <span class="error">&#91;"noreply"&#93;</span></font><font color="#6a8759">def recipients = msg.getRecipients(javax.mail.Message.RecipientType.TO)</font><font color="#6a8759">def filtered = recipients.findAll { emailAddress -></font><font color="#6a8759"> def emailAddressString = emailAddress.toString()</font><font color="#6a8759"> return domainWhiteList.any { domain -> emailAddressString.endsWith(domain) } && </font><font color="#6a8759"> !containsBlackList.any { value -> emailAddressString.contains(value) }</font>{color:#6a8759}}<br/>
<font color=""><font color="#6a8759"></font>msg.setRecipients(javax.mail.Message.RecipientType.TO, filtered as javax.mail.Address[])</font><font color="#6a8759">"""</font></p>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i><img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/davidvanlaatum.png" title="davidvanlaatum's avatar" /> davidvanlaatum:</i></summary>

<ul>
<li><i>Original <a class="original-jira-link" href="https://issues.jenkins.io/browse/JENKINS-43885?focusedId=345924&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-345924">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p>does it happen without the presend script?</p>
</pre>
</details>

</ul>
</details>

<p>does it happen without the presend script?</p>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i><img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/slide_o_mix.png" title="slide_o_mix's avatar" /> slide_o_mix:</i></summary>

<ul>
<li><i>Original <a class="original-jira-link" href="https://issues.jenkins.io/browse/JENKINS-43885?focusedId=363375&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-363375">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p>Waiting on user input, reopen with requested information.</p>
</pre>
</details>

</ul>
</details>

<p>Waiting on user input, reopen with requested information.</p>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i>svoelkel:</i></summary>

<ul>
<li><i>Original <a class="original-jira-link" href="https://issues.jenkins.io/browse/JENKINS-43885?focusedId=379039&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-379039">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p>I have a similar looking Stacktrace with Jenkins 2.190.1, email-ext 2.68.</p>

<p>We are using the <a href="https://wiki.jenkins.io/display/JENKINS/Email-ext+Recipes#Email-extRecipes-FilterRecipientsOnWhiteList" class="external-link" target="_blank" rel="nofollow noopener">Filter Recipients On White List</a> presend recipe. Without it, I am not getting the stacktrace.</p>

<p><a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=slide_o_mix" class="user-hover" rel="slide_o_mix">slide_o_mix</a>: should I attach the stacktrace here, or do you prefer a new issue?</p>
</pre>
</details>

</ul>
</details>

<p>I have a similar looking Stacktrace with Jenkins 2.190.1, email-ext 2.68.</p>

<p>We are using the <a href="https://wiki.jenkins.io/display/JENKINS/Email-ext+Recipes#Email-extRecipes-FilterRecipientsOnWhiteList" class="external-link" target="_blank" rel="nofollow noopener">Filter Recipients On White List</a> presend recipe. Without it, I am not getting the stacktrace.</p>

<p><a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=slide_o_mix" class="user-hover" rel="slide_o_mix">slide_o_mix</a>: should I attach the stacktrace here, or do you prefer a new issue?</p>

**Comment by jenkins-infra-bot** [comment]
<details><summary><i><img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/slide_o_mix.png" title="slide_o_mix's avatar" /> slide_o_mix:</i></summary>

<ul>
<li><i>Original <a class="original-jira-link" href="https://issues.jenkins.io/browse/JENKINS-43885?focusedId=379061&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-379061">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p>A new issue please.</p>
</pre>
</details>

</ul>
</details>

<p>A new issue please.</p>

**Comment by jenkins-infra-bot** [comment]
<i>[Original `duplicates` from Jira: <a class="jira-relationship" href="https://issue-redirect.jenkins.io/issue/43869">JENKINS-43869</a>]</i>
